kafka-rest/src/main/java/io/confluent/kafkarest/config/SchemaRegistryConfig.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2021 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.kafkarest.config;
+
+import static java.util.Objects.requireNonNull;
+
+import io.confluent.kafka.schemaregistry.ParsedSchema;
+import io.confluent.kafka.serializers.AbstractKafkaSchemaSerDeConfig;
+import io.confluent.kafka.serializers.subject.strategy.SubjectNameStrategy;
+import java.util.Map;
+import org.apache.kafka.common.config.ConfigDef;
+
+final class SchemaRegistryConfig extends AbstractKafkaSchemaSerDeConfig {
```

Comment:
doesn't seem like an inheritance relationship is logically correct here

Reply:
That's more a fault of `AbstractKafkaSchemaSerDeConfig` which should really be named `SchemaRegistryConfig` and not be abstract. That class only defines the SR configs, and it is what is used internally in `SchemaRegistryClient`.

Reply:
🤦